src/cs/ubc/ca/graphTeX/tokenize/Tokenizer.java

```diff
@@ -32,21 +31,16 @@ private void tokenize (){
         String tokenizedProgram = program;
         tokenizedProgram = tokenizedProgram.replace("\n","");
         tokenizedProgram = tokenizedProgram.replaceAll("([0-9]+)","_$1_");
-        System.out.println(program);
 
         for (String s : literals){
-//            tokenizedProgram = tokenizedProgram.replaceAll(s,"_"+s+"_");
             tokenizedProgram = tokenizedProgram.replace(s,"_"+s+"_");
-            System.out.println(tokenizedProgram);
         }
-        tokenizedProgram = tokenizedProgram.replaceAll("__","_");
-        tokenizedProgram = tokenizedProgram.replaceAll(" ","");
```

Comment:
If we're getting rid of this line, should we be trimming node names in the parser in case there are leading spaces?

Reply:
we probably should, but we can probably do that in the parser once nodes are implemented in it